Assets/MixedRealityToolkit.Tests/PlayModeTests/PlayModeTestUtilities.cs

```diff
@@ -29,12 +29,59 @@ namespace Microsoft.MixedReality.Toolkit.Tests
 {
     public class PlayModeTestUtilities
     {
-
         // Unity's default scene name for a recently created scene
         const string playModeTestSceneName = "MixedRealityToolkit.PlayModeTestScene";
 
         private static Stack<MixedRealityInputSimulationProfile> inputSimulationProfiles = new Stack<MixedRealityInputSimulationProfile>();
 
+        /// <summary>
+        /// The default number of frames that elapse for each test hand movement.
+        /// Intentionally a smaller number to keep tests fast.
+        /// </summary>
+        private const int HandMoveStepsDefault = 5;
+
+        /// <summary>
+        /// The default number of frames that elapse when in slow test hand mode.
+        /// See UseSlowTestHand for more information.
+        /// </summary>
+        private const int HandMoveStepsSlow = 60;
+
+        /// <summary>
+        /// If true, the hand movement test steps will take a longer number of frames. This is especially
+        /// useful for seeing motion in play mode tests (where the default smaller number of frames tends
+        /// to make tests too fast to be understandable to the human eye). This is false by default
+        /// to ensure that tests will run quickly in general, and can be set to true manually in specific
+        /// test cases using the example below.
+        /// </summary>
+        /// <example> 
+        /// <code>
+        /// [UnityTest]
+        /// public IEnumerator YourTestCase()
+        /// {
+        ///     PlayModeTestUtilities.UseSlowTestHand = true;
+        ///     ...
+        ///     PlayModeTestUtilities.UseSlowTestHand = false;
+        /// }
+        /// </code>
+        /// </example>
+        public static bool UseSlowTestHand = false;
```

Comment:
This seems a little dangerous if someone forgets to reset (exception thrown etc). Should PlayModeTestUtilities.Setup auto set this to false?

I'm not sure if PlayModeTestUtilities static variables are reloaded between tests cases?

Reply:
> Cool addition. NumHandSteps strongly controls the execution time of most tests I think.
> 
> Few minor questions to consider before going in

Yep - the intent here was to push things to a smaller count of frames, to speed up the test execution by default. Ideally we want people to get used to smaller amounts of wasted frames, and not get used to having large amount of buffer.

I'll clean up the stuff here in bit, thanks for the review!